main/observing/guider.py

```diff
@@ -157,8 +158,8 @@ def guiding_procedure(self, image_path):
             if not star:
                 logging.warning('Guider could not find a suitable guide star...waiting for next image to try again.')
                 continue
-            x_0 = 250
-            y_0 = 250
+            x_0 = self.config_dict.guider_max_move / self.config_dict.plate_scale
```

Comment:
change this to 
`x_0 = y_0 = self.config_dict.guider_max_move / self.config_dict.plate_scale`

Reply:
Oh cool, didn't know you could do that!